zerver/lib/import_realm.py

```diff
@@ -1373,6 +1394,16 @@ def do_import_realm(import_dir: Path, subdomain: str, processes: int = 1) -> Rea
 
     sender_map = {user["id"]: user for user in data["zerver_userprofile"]}
 
+    # TODO: de-dup how we read these json files.
+    attachments_file = os.path.join(import_dir, "attachment.json")
+    if not os.path.exists(attachments_file):
+        raise Exception("Missing attachment.json file!")
+
+    with open(attachments_file, "rb") as f:
+        attachment_data = orjson.loads(f.read())
+
+    map_messages_to_attachments(attachment_data)
```

Comment:
I think it's worth mentioning here in a comment the dependence that it's crucial for this to run before `import_message_data`, since it's very not obvious without looking into the internals of these functions

Reply:
Yes, i agree